README.md

```diff
@@ -17,7 +17,7 @@ Prerequisites
 Complete the installation:
 1. Find Codewind for VS Code in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) or by searching for `Codewind` in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
 2. Open the **Codewind** view in the Explorer view group or enter `Focus on Codewind View` into the [**Command Palette**](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette). If you do not see the Codewind view in either the Explorer view or the **Command Palette**, the extension did not install correctly.
-3. Codewind requires the installation of additional Docker images to run. Choose **Install** when prompted to complete the installation. The installation may take a few minutes to complete. Codewind creates a folder called `codewind-workspace` within your home directory to contain your projects. On Windows, this is the `C:\codewind-workspace` directory. When the installation is complete, you can open the `codewind-workspace` folder or a project within the workspace as your VS Code workspace. The tools offer to open the workspace for you if it’s not open already.
+3. Codewind requires the installation of additional Docker images to run. Choose **Install** when prompted to complete the installation. The installation may take a few minutes to complete. Before 0.6.0, projects had to be created and saved in a fixed location on users' disk called, `codewind-workspace`. 0.6.0 allows you to have your project created and saved anywhere and also removes the need for your project to be in a container. 
```

Comment:
I don't think we need to talk about the old behaviour here. READMEs generally just talk about what the product does right now. I would remove everything after "the installation may take a few minutes to complete".

Reply:
Cool, thanks, and will do!